docs/technical-reference.md

```
@@ -0,0 +1,1825 @@
+    
+    RestGET --> HandleRestFailures
+    RestGET --> IsModArchResponse
+    
+    APIIndex --> UseAPIState
+    APIIndex --> APITypes
+    
+    style APIIndex fill:#e1f5fe
+    style RestGET fill:#f3e5f5
+    style UseAPIState fill:#e8f5e8
+```
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)